rs/crypto/tests/canister_threshold_schnorr_verify_combined_sig.rs

```diff
@@ -156,6 +156,7 @@ fn should_fail_when_key_internal_transcript_raw_switched() {
             ThresholdSchnorrSigInputs::new(
                 inputs.derivation_path(),
                 inputs.message(),
+                None,
```

Comment:
IIUC, all tests for the taproot input are at the crypto lib level. Would it make sense to at least add a smoke test in some(/all?) `rs/crypto/tests/canister_threshold_schnorr_` to ensure that the taproot parameter is not disregarded/swallowed on the way down to the crypto lib?

Reply:
Yes I'll add some tests at this level

Reply:
Done